api/v1/sriovnetworknodestate_types.go

```diff
@@ -82,6 +82,7 @@ type VirtualFunction struct {
 	Vlan       int    `json:"Vlan,omitempty"`
 	Mtu        int    `json:"mtu,omitempty"`
 	VfID       int    `json:"vfID"`
+	VdpaType   string `json:"vdpaType,omitempty"`
```

Comment:
i guess we should hold off merging this until we got some logic to populate this field ?

Reply:
I will submit commits which relies on this change soon. Also this field is a part of internal API which is used only by the operator itself.

Reply:
ack.